lib/basket.rb

```diff
@@ -0,0 +1,30 @@
+class Basket
+
+  attr_reader :items
+
+    def initialize
+      @basket = []
+    end
+
+    def view
+      @basket
+    end
+
+    def price
+      # @price = @basket.inject(:+)
+    end
+
+    def order(item)
+      @basket << item
+    end
+
+    # def order(item, quantity)
+    #
+    #   @price = items.fetch(item)
+    #   @items[item]
+    #   puts "Thank you for ordering #{quantity} x #{item}, costing £#{list_of_items.fetch(item)} each"
```

Comment:
Metrics/LineLength: Line is too long. [103/80]<br>Style/AsciiComments: Use only ascii symbols in comments.